src/envoy/mixer/integration_test/repositories.bzl

```diff
@@ -76,16 +73,19 @@ DESCRIPTOR_FILE_GROUP = [
     "mixer/v1/config/descriptor/value_type.proto",
 ]
 
-gogoslick_proto_library(
-    name = "mixer/v1/config",
+# gogoslick_proto_compile cannot be used here. it generates Equal, Size, and
```

Comment:
Do you mean `gogoslick_proto_library`? It isn't clear to me why this causes compilation issues though. Does mixer do this too?